debian/rules

```diff
@@ -11,7 +11,9 @@ DISABLE_JEMALLOC_ARCH_LIST := hppa m68k
 DEB_HOST_ARCH ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
 
 # Set local state dir for FHS
-LOCAL_CONFIGURE_FLAGS = --localstatedir=/var/lib --libdir=/usr/lib
+LOCAL_CONFIGURE_FLAGS  = --localstatedir=/var/lib --libdir=/usr/lib
+LOCAL_CONFIGURE_FLAGS += --with-sphinx-build=sphinx-build-3
+LOCAL_CONFIGURE_FLAGS += RST2MAN=/usr/share/docutils/scripts/python3/rst2man
```

Comment:
The "/usr/bin/rst2man" command is a symlink managed by update-alternatives, it will provide rst2man regardless of which version is installed.  Would using the normal "rst2man" command in path suffice, if you control its availability with your build depends?

Reply:
Same reason as rst2man, the build failed because it was missing the "normal" command so I had to look at the package's contents to find what to use.

Not sure what went wrong.